packages/next/src/server/base-server.ts

```diff
@@ -2413,13 +2413,18 @@ export default abstract class Server<ServerOptions extends Options = Options> {
         delete headers[NEXT_CACHE_TAGS_HEADER]
       }
 
+      const nullBodyStatus = [101, 204, 205, 304]
```

Comment:
Is it possible to hoist the `nullBodyStatus` to the top of the module, to prevent re-creating the object multiple times?

Also, `Set.prototype.has` is way faster than `Array.prototype.includes`.